src/main/java/com/band/api/repository/UserEndorsementDao.java

```diff
@@ -0,0 +1,28 @@
+package com.band.api.repository;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Builder
+@Getter
+@Table(name = "user_endorsement", catalog = "band")
+public class UserEndorsementDao {
+    @Id
```

Comment:
Does this need a generation strategy?

Reply:
Probably but none of these tables are connected as is so there are a lot of problems, if this is the only problem we have other problems.